_redirects

```diff
@@ -61,3 +61,5 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 
 # Redirect Crowdsale API to deprecation page
 /contracts/3.x/api/crowdsale /contracts/3.x/crowdsales 301
+/contracts/3.x/api/drafts /contracts/3.x/drafts 301
+/contracts/3.x/api/lifecycle /contracts/3.x/api/utils 301
```

Comment:
I'm wondering if it might be best to redirect the unversioned `/contracts/api/drafts` (etc.) as opposed to the 3.x URLs specifically. Because the 3.x URLs will not exist anywhere. The only way someone might reach them is if they go through the unversioned one which redirects to latest. By redirecting the unversioned URL directly we're doing it in a more future-proof way, otherwise we'll have to keep adding `4.x` redirects and so on.

Reply:
That could work, yes! We'd have to make sure we don't deploy such a rule before 3.0 is actually out however.

I'd suggest continuing with the current approach, and then removing the versioning once 3.0 is out and its docsite live. WDYT?

Reply:
Sure. But let's do it as soon as possible once it's live.